cafebabel/static/js/textarea-change.js

```diff
@@ -0,0 +1,27 @@
+/* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
+function textAreaAutoResize(element, event, handler) {
```

Comment:
I don't get why we need that function instead of directly calling `addEventListener` below?

Reply:
OH, I think it's because in the copied version there was a switch between `addEventListener` and `attachEvent`. I guess there's no more reason now.

Reply:
Feel free to edit this yourself please @vinyll @davidbgk . I don't know the difference between js versions. Just make it consistant with the rest we've been doing: ES6, right?

Reply:
I am sure you are right, and you see in 2 sec if we need this function or if we don't